### PR TITLE
Add use cases to get the zip download limit and embargo duration in months

### LINKED ISSUE #84: Add use case to read settings required by the SPA (Embargo enabled and zip download limit)
**Overview of the Feature Request**

We need new use case logic to allow reading the status of these settings. This is necessary for the SPA to show the related functionality.

**What kind of user is the feature intended for?**

Package consumer

**What inspired the request?**

- https://github.com/IQSS/dataverse/issues/9834

**What existing behavior do you want changed?**

None

**Any brand new behavior do you want to add to Dataverse?**

New use case for reading settings

**Any open or closed issues related to this feature request?**

- https://github.com/IQSS/dataverse/issues/9834

### PR DESCRIPTION
**Pull request by GPortas**
## What this PR does / why we need it:

We need these new cases since they are necessary for the SPA to show the related functionality.

## Which issue(s) this PR closes:

- Closes #84 

## Related Dataverse PRs:

- Depends on https://github.com/IQSS/dataverse/pull/9853
- Depends on https://github.com/IQSS/dataverse/pull/9881

## Special notes for your reviewer:

Tests related to files are failing, since the Dataverse branch they are pointing to does not have the files API extension. Therefore, this PR must wait for the PR https://github.com/IQSS/dataverse/pull/9853 to be merged and https://github.com/IQSS/dataverse/pull/9881 to be updated with develop.

## Suggestions on how to test this:

- Visual inspection of the code
- Check unit and integration tests execution in GitHub actions

## Is there a release notes update needed for this change?:

Add GetZipDownloadLimit and GetMaxEmbargoDurationInMonths use cases

## Additional documentation:

None

### CODE REVIEW

File: test/integration/info/DataverseInfoRepository.test.ts [hunk 1]
```diff
@@ -1,14 +1,51 @@
...
+      // The setting exists
+      const testMaxEmbargoDurationInMonths = 12;
+      await setMaxEmbargoDurationInMonthsViaApi(testMaxEmbargoDurationInMonths)
+        .then()
+        .catch(() => {
+          fail('Test getMaxEmbargoDurationInMonths: Error while setting :MaxEmbargoDurationInMonths');
+        });
+      const actual = await sut.getMaxEmbargoDurationInMonths();
+      assert.match(actual, testMaxEmbargoDurationInMonths);
+    });
```
Comment (MellyGray) [inline test/integration/info/DataverseInfoRepository.test.ts]: I think this test could be divided into 2 different tests

test('should return error if the setting does not exist'
test('should return duration if the setting exists'

**Review by MellyGray — COMMENTED** [review]
Looks good. Just a comment about splitting a test

Hopefully the dataverse branches this PR depends on get merged soon

**Comment by GPortas** [comment]
@MellyGray 

There are use cases whose integration tests are failing, that is normal, since the Dataverse branch used in this PR does not include all endpoints due to Dataverse PRs not yet merged.

**Comment by MellyGray** [comment]
> There are use cases whose integration tests are failing, that is normal, since the Dataverse branch used in this PR does not include all endpoints due to Dataverse PRs not yet merged.

@GPortas I know, but should the integration tests point to DATAVERSE_IMAGE_TAG=9880-info-zip-download-limit-embargo ? branch 9880 is already merged into develop so maybe you can point to the develop image tag so when the other files PR get merged into develop the integration tests work

I'm ready to approve this, just wanted to comment that

**Comment by GPortas** [comment]
@MellyGray 

I agree. In fact, in an ideal scenario the develop branch of this repository should always point to dataverse develop branch image (unstable tag).

Updating.

**Review by MellyGray — APPROVED** [review]
LGTM 👍

**Pull request merged**